bundles/core/org.eclipse.smarthome.core.voice/src/main/java/org/eclipse/smarthome/core/voice/internal/VoiceConsoleCommandExtension.java

```diff
@@ -44,7 +44,8 @@ public VoiceConsoleCommandExtension() {
 
     @Override
     public List<String> getUsages() {
-        return Arrays.asList(new String[] { buildCommandUsage(SUBCMD_SAY + " <text>", "speaks a text"),
+        return Arrays.asList(new String[] {
+                buildCommandUsage(SUBCMD_SAY + " <sink> <text>", "speaks a text through the given audio sink"),
```

Comment:
This makes the command line usage pretty awkward - I intentionally did it this way to keep it simple. The user can anyhow configure his defaultSink, which is used by the console commands.